package.json

```diff
@@ -16,29 +20,44 @@
     "Jeff Whelpley <jeff@gethuman.com>"
   ],
   "devDependencies": {
-    "@types/es6-promise": "0.0.29",
-    "@types/jasmine": "^2.2.31",
-    "@types/node": "^6.0.33",
-    "connect-livereload": "^0.5.4",
-    "del": "^2.2.1",
-    "express": "^4.14.0",
-    "gulp": "^3.9.1",
-    "gulp-jasmine": "^2.4.0",
-    "gulp-livereload": "^3.8.1",
-    "gulp-rename": "^1.2.2",
-    "gulp-replace": "^0.5.4",
-    "gulp-size": "^2.1.0",
-    "gulp-tslint": "^6.0.2",
-    "gulp-typedoc": "^2.0.0",
-    "gulp-typescript": "^2.13.6",
-    "gulp-uglify": "^2.0.0",
-    "open": "0.0.5",
-    "serve-static": "^1.11.1",
-    "tslint": "^3.14.0",
-    "typedoc": "^0.4.4",
-    "typescript": "2.1.0-dev.20160801"
+    "@angular/animations": "^4.1.3",
+    "@angular/common": "^4.1.3",
+    "@angular/compiler": "^4.1.3",
+    "@angular/core": "^4.1.3",
+    "@angular/http": "^4.1.3",
+    "@angular/platform-browser": "^4.1.3",
+    "@angular/platform-browser-dynamic": "^4.1.3",
+    "@angular/platform-server": "^4.1.3",
+    "@types/express": "^4.0.35",
+    "@types/jasmine": "^2.5.47",
+    "@types/node": "^7.0.22",
+    "@types/uglify-js": "^2.6.28",
+    "express": "^4.15.3",
+    "http-server": "^0.10.0",
+    "jasmine": "^2.6.0",
+    "jasmine-spec-reporter": "^4.1.0",
+    "protractor": "~5.1.2",
+    "reflect-metadata": "^0.1.10",
+    "rxjs": "^5.4.0",
+    "webdriver-manager": "^12.0.6",
+    "webpack": "^2.6.1",
+    "zone.js": "^0.8.11"
+  },
+  "dependencies": {
+    "uglify-js": "^3.0.11"
```

Comment:
I was trying to avoid actual dependencies, but can't avoid this one without forcing the user to do something weird. This way, the inline preboot code can be minified behind the scenes.